src/main/java/com/iopipe/IOpipeExecution.java

```diff
@@ -615,26 +615,44 @@ final RemoteRequest __buildRequest()
 			{
 				CustomMetric metric = custmetrics[i];
 				
+				// Check that the name is in the limit
 				String xname = metric.name();
-				if (IOpipeExecution.__isNameInLimit(xname))
+				if (!IOpipeExecution.__isNameInLimit(xname))
 				{
-					gen.writeStartObject();
-					
-					gen.write("name", xname);
-					
-					if (metric.hasString())
-						gen.write("s", metric.stringValue());
-					if (metric.hasLong())
-						gen.write("n", metric.longValue());
-					
-					gen.writeEnd();
-				}
-				
-				// Emit warning
-				else
 					_LOGGER.warn("Metric exceeds the {} codepoint limit and " +
 						"will not be reported: {}",
 						IOpipeConstants.NAME_CODEPOINT_LIMIT, xname);
+					continue;
+				}
+				
+				// Check if the value is in range
+				String svalue;
+				if (metric.hasString())
+				{
+					svalue = metric.stringValue();
+					
+					if (!IOpipeExecution.__isValueInLimit(svalue))
+					{
+						_LOGGER.warn("Metric value exceeds the {} codepoint " +
```

Comment:
string length? value length? Maybe `codepoint limit` is clear enough for Java devs?

Reply:
Perhaps this can be `codepoint length limit` instead. In Java, codepoints are distinct from characters although they are similar.